amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -284,6 +284,16 @@ export class TableDetail extends React.Component<
                     <div className="section-title title-3">Frequent Users</div>
                     <FrequentUsers readers={data.table_readers} />
                   </section>
+                  {data.programmatic_descriptions.left.map((d) => (
+                    <EditableSection title={d.source} readOnly>
```

Comment:
Curious if React doesn't complain about this component not having a key attribute.

Reply:
It surprisingly did not -- at least in the build script -- but I will add them

Reply:
Will also pull this out into a helper method since it's the same logic 3 times.